src/coreclr/tools/aot/ILCompiler.Diagnostics/ILCompiler.Diagnostics.csproj

```diff
@@ -21,6 +21,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ILCompiler.TypeSystem\ILCompiler.TypeSystem.csproj" />
+    <ProjectReference Include="..\..\..\..\libraries\System.Runtime.InteropServices\tests\Ancillary.Interop\Ancillary.Interop.csproj" />
```

Comment:
We should be able to remove these two ProjectReferences now that we're on the p4 SDK